src/expressions/functions/string.ts

```diff
@@ -96,9 +110,23 @@ export function right(original: Expr, length: Expr): CypherFunction {
  * @see [Cypher Documentation](https://neo4j.com/docs/cypher-manual/current/functions/string/)
  * @group Cypher Functions
  * @category String
+ * @deprecated Use {@link rtrim} instead
  */
-export function rTrim(original: Expr): CypherFunction {
-    return new CypherFunction("rTrim", [original]);
+export function rTrim(original: Expr | string, trimCharacter?: string | Expr): CypherFunction {
+    const normalizedOriginal = normalizeExpr(original);
+    const normalizedTrimCharacter = trimCharacter ? normalizeExpr(trimCharacter) : undefined;
+    return new CypherFunction("rTrim", filterTruthy([normalizedOriginal, normalizedTrimCharacter]));
```

Comment:
```suggestion
    return rtrim(original, trimCharacter);
```

Reply:
These are technically breaking, as the resulting cypher changes :see_no_evil: 

This may break tck tests in the https://github.com/neo4j/graphql library

Reply:
You're right I didn't notice!